exporters/otlp/otlplog/otlploggrpc/config.go

```diff
@@ -138,44 +51,6 @@ func WithHeaders(headers map[string]string) Option {
 	return nil
 }
 
-// WithTLSCredentials sets the gRPC connection to use creds.
-//
-// If the OTEL_EXPORTER_OTLP_CERTIFICATE or
-// OTEL_EXPORTER_OTLP_LOGS_CERTIFICATE environment variable is set, and
-// this option is not passed, that variable value will be used. The value will
-// be parsed the filepath of the TLS certificate chain to use. If both are
-// set, OTEL_EXPORTER_OTLP_LOGS_CERTIFICATE will take precedence.
-//
-// By default, if an environment variable is not set, and this option is not
-// passed, no TLS credentials will be used.
-//
-// This option has no effect if WithGRPCConn is used.
-func WithTLSCredentials(_ credentials.TransportCredentials) Option {
-	// TODO: implement.
-	return nil
-}
-
-// WithServiceConfig defines the default gRPC service config used.
-//
-// This option has no effect if WithGRPCConn is used.
-func WithServiceConfig(serviceConfig string) Option {
-	// TODO: implement.
-	return nil
-}
-
-// WithDialOption sets explicit grpc.DialOptions to use when establishing a
-// gRPC connection. The options here are appended to the internal grpc.DialOptions
-// used so they will take precedence over any other internal grpc.DialOptions
-// they might conflict with.
-// The [grpc.WithBlock], [grpc.WithTimeout], and [grpc.WithReturnConnectionError]
-// grpc.DialOptions are ignored.
-//
-// This option has no effect if WithGRPCConn is used.
-func WithDialOption(_ ...grpc.DialOption) Option {
```

Comment:
I think we should consider keeping DialOptions: https://github.com/open-telemetry/opentelemetry-go/issues/5248#issuecomment-2082745568